src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java

```diff
@@ -26,7 +26,6 @@
 import uk.gov.hmcts.reform.civil.model.docmosis.sealedclaim.Representative;
 import uk.gov.hmcts.reform.civil.documentmanagement.model.CaseDocument;
```

Comment:
Wouldnt there be a Functional Test required for this Change...?

Reply:
you're right!! I refactored the documentControllerTest class as you suggest